.github/workflows/build.yml

```diff
@@ -55,8 +55,10 @@ jobs:
         echo ============================
         qmake -r PythonQt.pro CONFIG+=release CONFIG+=sanitizer CONFIG+=sanitize_undefined \
         PYTHON_VERSION=$(python3 --version | cut -d " " -f 2 | cut -d "." -f1,2) \
-        PYTHON_DIR=$(which python3 | xargs dirname | xargs dirname)    
-        make -j 2 && make check TESTARGS="-platform offscreen"
+        PYTHON_DIR=$(which python3 | xargs dirname | xargs dirname)
+        make -j 2
+        UBSAN_OPTIONS="halt_on_error=1" ASAN_OPTIONS="detect_stack_use_after_return=1:fast_unwind_on_malloc=0" \
```

Comment:
I have no experience with UBSAN, will rely on your expertise here.

Unrelated note: I'm actually now thinking that these container based test on Ubuntu can be replaced tests based on `install-qt-action` as the macos and Windows tests - this makes it better configurable. Maybe only retain the CentOS one to be sure that it still works there...

Reply:
I advise to save a previous and the latest Ubuntu LTS build in CI to test default build with them. Just to be sure that everything woks fine on typical developer machine.